diff --git a/src/ht_server.c b/src/ht_server.c
index e759523..c2e6548 100644
--- a/src/ht_server.c
+++ b/src/ht_server.c
@@ -52,9 +52,8 @@ static void send_alloc_stat(const ht_alloc_stat_t *stat, void *arg)
 			void *calibrated = (void *)((uintptr_t)bt -
 						    (uintptr_t)info.dli_fbase);
 
-			ptr += sprintf(ptr, "addr2line -e %s %p | %s\n",
-				       info.dli_fname, calibrated,
-				       info.dli_sname);
+			ptr += sprintf(ptr, "%p (%p) | %s of %s\n", calibrated,
+				       bt, info.dli_sname, info.dli_fname);
 		}
 	}
 
